src/borg/cache.py

```diff
@@ -921,6 +937,15 @@ def wipe_cache(self):
         self.chunks = ChunkIndex()
         with SaveFile(os.path.join(self.path, files_cache_name()), binary=True):
             pass  # empty file
+        self.recreate_cache_config()
+
+    def recreate_cache_config(self, delete_existing=False):
```

Comment:
maybe call this `create...` - it only mutates to the more special `recreate_...` in you give `delete_existing=True`.